pkg/controller/provider/model/vsphere/model.go

```diff
@@ -23,8 +28,8 @@ type Base struct {
 	Name string `sql:""`
 	// Parent
 	Parent string `sql:"index(a)"`
-	// Annotations
-	Annotations string `sql:""`
```

Comment:
Annotations did not end up being generally useful.
Replaced with `concerns` on VM.